Code/Framework/AzQtComponents/AzQtComponents/Gallery/CardPage.cpp

```diff
@@ -70,6 +70,16 @@ header->setHelpURL("https://o3de.org/docs/");
 // Clear the help url
 header->clearHelpURL();
 
+// Set header underline color to make some card visually popping
+header->setUnderlineColor(Qt::cyan);
+
+// Separator cards don't have a visually distinct contents area and can
+// be used for grouping other cards and widgets
+AzQtComponents* separatorCard;
+AzQtComponents::Card::applySeparatorStyle(separatorCard);
+auto* nestedCard = new AzQtComponents::Card();
+separatorCard->setContentWidget(nestedCard);
```

Comment:
Thanks for adding this to the gallery and these great comments and example usage 👍

Reply:
👍